src/utils/api.ts

```diff
@@ -0,0 +1,104 @@
+const apiUrl = "https://disease.sh/v3/covid-19/";
+
+async function request(endpoint: string) {
+  const response = await fetch(apiUrl + endpoint);
+  return response.json();
+}
```

Comment:
Interesting -- you annotated parameter types but function return types are not annotated, is this intended? I.e. I'd expect this to be something like

```ts
async function request(endpoint: string): JSON {
  const response = await fetch(apiUrl + endpoint);
  return response.json();
}
```

Reply:
TS newbie here T T